From 60c47e13a0aac3e52d652353a17aa23432bb08dc Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Thu, 13 Jun 2024 10:13:29 +0200
Subject: [PATCH] WAM: Fix outbound permission error

Fixes:

Fixes: Apr 11 22:05:24 qemux86-64 webos-connman-adapter[1266]: [] [pmlog] LS_REQUIRES_SECURITY {"CLIENT":"com.webos.settingsservice.client-1136","SERVICE":"com.webos.service.connectionmanager","CATEGORY":"/","METHOD":"getStatus"} Service security groups don't allow method call.

Might be a partial solution for YouTube not working in recent releases.
Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Inappropriate [LuneOS specific]

 files/sysbus/com.palm.webappmanager.perm.json | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/files/sysbus/com.palm.webappmanager.perm.json b/files/sysbus/com.palm.webappmanager.perm.json
index cea0eff..e42d6c4 100644
--- a/files/sysbus/com.palm.webappmanager.perm.json
+++ b/files/sysbus/com.palm.webappmanager.perm.json
@@ -30,7 +30,8 @@
     "com.webos.settingsservice.client-*": [
         "application.launcher",
         "application.operation",
-        "settings.query"
+        "settings.query",
+        "networkconnection.query"
     ],
     "com.webos.chromium.installablemanager.*": [
         "applicationinstall.management",
